src/app/Fake.Installer.Squirrel/Squirrel.fs

```diff
@@ -0,0 +1,142 @@
+    new StringBuilder()
+    |> StringBuilder.appendWithoutQuotes "--signWithParams=\""
+    |> StringBuilder.appendWithoutQuotes "/a"
+    |> StringBuilder.appendIfSome parameters.SigningKeyFile (sprintf "/f %s")
+    |> StringBuilder.appendIfSome parameters.SigningSecret  (sprintf "/p %s")
+    |> StringBuilder.appendWithoutQuotes "\""
+    |> StringBuilder.toText
+
+let internal buildSquirrelArgs parameters nugetPackage =
+    new StringBuilder()
+    |> StringBuilder.appendIfNotNullOrEmpty nugetPackage "--releasify="
+    |> StringBuilder.appendIfNotNullOrEmpty parameters.ReleaseDir "--releaseDir="
+    |> StringBuilder.appendIfSome parameters.LoadingGif (sprintf "\"--loadingGif=%s\"")
+    |> StringBuilder.appendIfSome parameters.SetupIcon (sprintf "\"--setupIcon=%s\"")
+    |> StringBuilder.appendIfTrue parameters.NoMsi "--no-msi"
+    |> StringBuilder.appendIfSome parameters.BootstrapperExe (sprintf "\"--bootstrapperExe=%s\"")
+    |> StringBuilder.appendIfSome parameters.SignExecutable (fun _ -> createSigningArgs parameters)
+    |> StringBuilder.toText
+
+module internal ResultHandling =
+    let (|OK|Failure|) = function
+        | 0 -> OK
+        | x -> Failure x
+
+    let buildErrorMessage = function
+        | OK -> None
+        | Failure errorCode ->
+            Some (sprintf "Squirrel reported an error (Error Code %d)" errorCode)
+
+    let failBuildIfSquirrelReportedError =
+        buildErrorMessage
+        >> Option.iter failwith
+
+/// Creates a Squirrel installer for given NuGet package
+/// Will fail if Squirrel terminates with non-zero exit code.
+///
+/// ## Parameters
+///
+///  - `setParams` - Function used to manipulate the default `SquirrelParams` value.
+///  - `nugetPackage` - The package to create an installer for
+///
+/// ## Sample usage
+///
+///     Target.create "CreatePackage" (fun _ ->
+///         Squirrel.releasify "./my.nupkg" (fun p -> { p with ReleaseDir = "./squirrel_release")
+///     )
+/// 
+/// ## Defaults for setParams
+///
+/// - `ReleaseDir` - `""`
+/// - `WorkingDir` - `None`
+/// - `BootstrapperExe` - `None`
+/// - `LoadingGif` - `None`
+/// - `SetupIcon` - `None`
+/// - `NoMsi` - `false`
+/// - `ToolPath` - The `squirrel.exe` path if it exists in a subdirectory of the current directory.
+/// - `TimeOut` - 10 minutes
+/// - `SignExecutable` - `None`
+/// - `SigningKeyFile` - `None`
+/// - `SigningSecret` - `None`
+let releasify (nugetPackage: string) (setParams: ReleasifyParams -> ReleasifyParams): unit =
+    use __ = Trace.traceTask "Squirrel" nugetPackage
+    let parameters = defaultParams.Value |> setParams
+    let args = buildSquirrelArgs parameters nugetPackage
+    Trace.tracefn "%s" args
+
+    let result =
+        Process.execSimple
+            (fun info -> { info with
+                            FileName = parameters.ToolPath
+                            WorkingDirectory = defaultArg parameters.WorkingDir "."
+                            Arguments = args})
```

Comment:
If `Squirrel.exe` is a .NET assembly we might need to wrap the call with `mono` here.

Reply:
It is a .NET assembly but it uses lot of executables that are windows-only and produce a windows-only installer. No multi platform story here ;)